pkg/ansible/proxy/proxy.go

```diff
@@ -72,6 +72,15 @@ func CacheResponseHandler(h http.Handler, informerCache cache.Cache, restMapper
 				break
 			}
 
+			// check if resource doesn't exist in watched namespaces
+			// if watchedNamespaces[""] exists then we are watching all namespaces
+			// and want to continue
+			_, allNsPresent := watchedNamespaces[""]
```

Comment:
Good call, I've updated the PR with a constant.

Reply:
I disagree, the `""` is the default kubernetes value for all namespaces. It would be more confusing to have our own constant. I would instead use their constant instead of the `""` but otherwise think this sense.